com.unity.multiplayer.mlapi/Tests/Runtime/AOI/RandomInterestKernel.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Generic;
+using MLAPI.Connection;
+using MLAPI.Interest;
+using UnityEngine;
+
+namespace MLAPI.RuntimeTests.AOI
```

Comment:
Do you think we should have a namespace just for AOI?  If so it would seem you'd also want to change `InterestTests.cs` below to be in that namespace?

Reply:
Perhaps, I tried to not touch the existing stuff.

Reply:
Moved the tests from "AOI" to "Interest" directory and changed namespace to "MLAPI.RuntimeTests". We can deal with the "Interest" namespace in a separate PR